taxcalc/functions.py

```diff
@@ -151,6 +151,9 @@ def AGI(_ymod1, c02500, c02700, e02615, c02900, e00100, e02500, XTOT,
     _ywossbe = e00100 - e02500
     _ywossbc = c00100 - c02500
 
+    if DSI:
+        XTOT = 0
```

Comment:
Link: http://www.irs.gov/publications/p501/ar02.html#en_US_2014_publink1000220868
Keyword: "Dependent not allowed a personal exemption."